Common/src/interface_interpolation/CInterpolator.cpp

```diff
@@ -87,7 +87,7 @@ void CInterpolator::Collect_VertexInfo(int markDonor, int markTarget,
 
   for (iVertex = 0; iVertex < MaxLocalVertex_Donor; iVertex++) Buffer_Send_GlobalPoint[iVertex] = -1;
 
-  for (iVertex = 0; iVertex < MaxLocalVertex_Donor*nDim; iVertex++) Buffer_Send_Coord[iVertex] = 0.0;
+  Buffer_Send_Coord.resize(nVertexDonor,nDim);
```

Comment:
If zero-init is needed you can do 
```suggestion
  Buffer_Send_Coord.resize(nVertexDonor,nDim) = su2double(0.0);
```

Reply:
Thanks for the hint! Zero-initialization is actually not needed here, we just write the coordinates into the buffer afterwards.